DataFormats/StdDictionaries/plugins/BuildFile.xml

```diff
@@ -0,0 +1,10 @@
+
+
+<use name="DataFormats/Common"/>
+<use name="FWCore/PluginManager"/>
+<use name="TrivialSerialisation/Common"/>
+
+<export>
+  <lib name="1"/>
+</export>
+
```

Comment:
Shouldn't this be a _plugin_, and not a _library_ ?

Then
```suggestion
<library file="modules.cc" name="DataFormatsStdDictionariesPlugins">
  <use name="DataFormats/Common"/>
  <use name="FWCore/PluginManager"/>
  <use name="TrivialSerialisation/Common"/>
  <flags EDM_PLUGIN="1"/>
</library>
```

Or maybe it could have `name="DataFormatsStdDictionariesTrivialSerialisation"`.